.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
.DS_Store is not platform agnostic and should be taken care of by your local .gitignore_global file.

Reply:
Fixed!